docs/reference/technology/mesh/quick_start_hw.md

```diff
@@ -45,6 +45,16 @@ The following table shows which development boards have been tested. It does not
 | Onsemi NCS36510 <span style='background-color: #5f5;'>(internal RF)</span> | | | |
 | NXP KW24D <span style='background-color: #5f5;'>(internal RF)</span> | | <span style='background-color: #f00;'>Yes **(4)**</span> | |
 
+#### File System
+
+Thread network stack can save network configurations settings to file system once:
+
+- You enable the file system as instructed in the [Mbed OS storage documentation](https://os.mbed.com/docs/latest/reference/storage.html).
```

Comment:
Do you have to do these in a particular order? Then it should be 1, 2, not bullets